product_docs/docs/pgd/5.6/nodes/subscriber_only/overview.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: An overview of Subscriber-only nodes
+---
+
+## Overview
+
+While many use cases rely on accessing a database node which can handle queries and updates, there are also use cases which only require access to a node that can handle read-only database queries.
+Read scaling like this, by moving the read-only traffic away from active database nodes in the cluster, can improve the performance of the core cluster, whilst making database access more widely available.
+
+The basic idea of subscriber-only nodes is to provide a read-only node that can be used to offload read-only queries from the main cluster. 
+How it connects to the other nodes in the cluster can be different from how other nodes connect to each other.
+The default topology of a PGD cluster is what we call a full mesh topology, where every node connects to every other node.
```

Comment:
> a full mesh topology

Which is essentially what we achieve with BDR (Bi-Directional Replication). I wonder if we should reference that here or not. I don't know if we intend to continue to use the term BDR for anything in the future.

Reply:
We reference BDR all over the place, though I would note that BDR does not automatically mean a fully meshed topology. That's a topology we chose.